Epam.TestAutomation/Epam.TestAutomation.BDD/Features/CookieBanner.feature

```diff
@@ -0,0 +1,13 @@
+﻿@CookieBanner
+Feature: Cookie Banner Test
+  As a user
+  I want to see and interact with the cookie banner
+  So that I can manage my cookie preferences
+
+  Scenario: Open the page and verify the cookie banner
```

Comment:
It is better to use module-submodule-purpose structure. Example - Landing page - Cookie banner - Check that cookie banner is present after navigation to landing page